### DISCUSSION TITLE
Data Sources hiding feature

### DISCUSSION THREAD

**Discussion by kochiops** [grafana/grafana#72581] [Data Sources]
There are numerous datasources in Grafana atm. It is desirable to have a feature to hide them from being available upon creating a dashboard. It would allow one to avoid a misselection of a data source with the same title as a DS variables. If it is not possible through UI, an option like "hide: true" in provisioning.yml would also work fine

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82491. Feel free to continue the discussion around this there. Thank you!
